openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -333,7 +333,8 @@ This endpoint is used by a Credential Issuer that is already interacting with an
 
 ## Credential Offer {#credential-offer}
 
-The Credential Issuer sends Credential Offer using an HTTP GET request or an HTTP redirect to the Wallet's Credential Offer Endpoint defined in (#client-metadata).
+The Credential Issuer makes a Credential Offer by directing the resource owner's
```

Comment:
@c2bo makes a good point at https://github.com/openid/OpenID4VCI/pull/380#pullrequestreview-2279672949 :

> While "resource owner's user agent" might be technically correct, it sounds a bit off to me - especially given, that we have not used the term resource owner once in the whole spec so far.

so I'll suggest this with the term "End-User" that is used quite a bit throughout the draft:

```suggestion
The Credential Issuer makes a Credential Offer by directing the End-User's
```